cosmos/dbt/graph.py

```diff
@@ -137,15 +135,13 @@ def __init__(
         render_config: RenderConfig = RenderConfig(),
         execution_config: ExecutionConfig = ExecutionConfig(),
         profile_config: ProfileConfig | None = None,
-        dbt_cmd: str = get_system_dbt(),
```

Comment:
@tatiana you removed `dbt_cmd` from `__init__`, I think the example usage in the `DbtGraph` docstring needs to be updated now.

Reply:
Well observed, @draskell - fixing this now!